charts/airbyte/values.yaml

```diff
@@ -27,6 +27,15 @@ serviceAccount:
   annotations: {}
   name: airbyte-admin
 
+## Kubernetes Authentication
+## Authentication method used by airbyte pods requiring cluster access, eg. worker/pod-sweeper
+## @param kubernetesAuth.tryKubeConfig if true, will try to use kube config mounted inside the pod (default: true)
+## @param kubernetesAuth.tryServiceAccount if true, will try to use serviceAccount credentials from serviceAccount.name (default: false)
+##
+kubernetesAuth:
+  tryKubeConfig: true
```

Comment:
what does `tryKubeConfig` do?

Reply:
it tells you in the param section a couple of lines up:
```
## @param kubernetesAuth.tryKubeConfig if true, will try to use kube config mounted inside the pod (default: true)
```
I defaulted this to `true` as this is what seemed to be the way airbyte wanted to auth against k8s, given the errors in the logs

Reply:
Thanks for the fabric link. I understand this now. What do you think about removing the try kube config option and switching on the service account? I don't think a user would want to want both to be set to true/false so this seems like a clearer configuration option to me. We can have this tryServiceAccount variable be the switch. What do you think?

Can we rename `tryServiceAccount` to `useServiceAccount`? That seems clearer to me.

Reply:
> Thanks for the fabric link. I understand this now. What do you think about removing the try kube config option and switching on the service account? I don't think a user would want to want both to be set to true/false so this seems like a clearer configuration option to me. We can have this tryServiceAccount variable be the switch. What do you think?
Yeah, sounds good - I'll make the changes - I could not think of a use-case for having kube config mounted inside a k8s pod - but did not want to deviate too much, but I agree - it doesn't make sense - I also think that `useServiceAccount` should default to true as, again, I can't see many scenarios where this is not the desired behaviour?

> 
> Can we rename `tryServiceAccount` to `useServiceAccount`? That seems clearer to me.
Yep, will do